docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -116,7 +116,24 @@ Include it in your module file like this:
 .. code-block:: python
 
     #!/usr/bin/python
-    # Copyright header....
+    # Copyright © 2017 [REPLACE THIS]
+    # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+.. note::
+
+   - Every file with a copyright header must include the original author.
+     Major additions to the module are allowed to add an additional copyright
+     line. This is especially true of rewrites, but original authorship
+     copyright messages should be preserved. If applicable, contributors may
+     have their rights assigned to their employer for example: "Copyright ©
+     2017 Red Hat, Inc."
```

Comment:
Instead of talking about copyright assignment, maybe just talk about copyright holder?  Or maybe we don't want to talk about rights and assignment and such at all?  We decided that we're going to start using "Copyright (C) 2017 Ansible Project", for instance.  Ansible Project isn't a legal entity so it seems to not be about copyright holder...